platform.txt

```diff
@@ -6,7 +6,7 @@
 # https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5---3rd-party-Hardware-specification
 
 name=ESP8266 Modules
-version=2.5.0
+version=2.4.0-post
```

Comment:
According to semver, 2.4.0-post is something *before* 2.4.0.

It might be 2.4.1-pre if we intend to merge this before 2.4.1?